src/components/BookForm.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+
+const BookForm = () => {
+  const [inputText, setInputText] = useState({
+    title: '',
+  });
+
+  const onChange = (e) => {
+    setInputText({
+      ...inputText,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  return (
+    <div className="form">
+      <h1 className="newBook">ADD NEW BOOK</h1>
+      <input type="text" name="bookTitle" onChange={onChange} required placeholder="Book Title" />
+      <input type="text" name="bookAuthor" onChange={onChange} required placeholder="Name of the Author" />
+      <select name="bookGenre" className="bookGenreList">
+        <option value="Action">Action</option>
+        <option value="Science Fiction">Science Fiction</option>
+        <option value="Economy">Economy</option>
+        <option value="Others">Others</option>
+      </select>
+      <button type="submit" className="bookAddButton">Add Book</button>
+    </div>
```

Comment:
- [ ] Dear @PrangonGhose, Impressive job you have done already, But for the form component we need to return a `form` element (I see you use a `div` with `className-'form' `). using the form component have lots of advantage when it comes to accessibility and form submission. 
> you might wonder to use JavaScript for the form submission but still using a form tag makes it easy for browsers when it comes to semantics and it will also help you use events like `submit` to validate your form.